content-src/components/LinkMenuButton/LinkMenuButton.scss

```diff
@@ -6,7 +6,7 @@
   width: $link-menu-button-size;
   height: $link-menu-button-size;
   background-color: $white;
-  background-image: url('img/glyph-more-16.svg');
+  background-image: url($image-path + 'glyph-more-16.svg');
```

Comment:
Nit: Looks like some inconsistent interpolation stuffs between this file and the previous ActivityFeed.scss file:

| FILE | CODE |
|:----|:----|
| ActivityFeed/ActivityFeed.scss | `url('#{$image-path}glyph-bookmark-16-blue.svg')`
| LinkMenuButton/LinkMenuButton.scss | `url($image-path + 'glyph-more-16.svg')`

May be a bit cleaner to stick to one style (either `'#{$foo}bar'` or `$foo + 'bar'`).
Not sure if sass-lint has rules for this. I tried looking at https://github.com/sasstools/sass-lint/tree/master/docs/rules but nothing immediately jumps out.